packages/sync-actions/test/product-sync-prices.spec.js

```diff
@@ -595,7 +758,7 @@ describe('Actions', () => {
                 fractionDigits: 2,
               },
               id: '38a9667a-976d-4bcc-8d2a-b04701e41f18',
-              custom: {
+              customField: {
```

Comment:
This looks wrong, please revert this change.

Reply:
done